harness/determined/experimental/client.py

```diff
@@ -248,15 +248,19 @@ def list_experiments(
 def create_user(
     username: str, admin: bool, password: Optional[str] = None, remote: bool = False
 ) -> User:
-    """Create an user with username and password, admin.
+    """Creates a user.
```

Comment:
Nice. Thanks for this change!

Reply:
Actually... I think the docstring for `Determined.create_user` should just be copy/pasted here. This is the method our system's users are going to be using probably 95% of the time (this one, not `Determined.create_user`).

I _hate_ that this has to be done, but I don't see it as about to change anytime soon.

Reply:
I still hate this, and I hate even more that I'm leaving another comment. The docstrings aren't quite the same, and I think the one in `Determined` is slightly better. I'm sorry for this.